.github/workflows/link-validator.yml

```diff
@@ -0,0 +1,38 @@
+name: Link Validator
+
+on:
+  pull_request_target:
+    branches: [master, main]
```

Comment:
@ennru if I understand correctly from different PR's this is not the right way, right?

Reply:
Hm, you looked at an outdated version. I've updated this to `pull_request`.

Reply:
oh, weird, sorry